lib/server/vendor/coinpayments.js

```diff
@@ -0,0 +1,12 @@
+const Coinpayments = require('coinpayments');
+const log = require('../../logger');
+
+const options = {
+  key: process.env.CP_PUBLIC_KEY,
+  secret: process.env.CP_PRIVATE_KEY,
+  autoIpn: process.env.NODE_ENV === 'production' ? false : true
+}
+
+const client = new Coinpayments(options);
```

Comment:
same feedback as in service-storage-models... in fact maybe you can reference service-storage-models' copy instead of duplicating?

Reply:
I considered exporting Coinpayments client on the service-storage-models index. Still an option?